README.md

```diff
@@ -1,18 +1,27 @@
 # RAPIDS PEP517 build backend
 
 `rapids-build-backend` is an adapter around PEP517 builders that provides support for key RAPIDS requirements.
-It currently support `scikit-build-core` and `setuptools` as the wrapped builder.
+
 The package's primary purpose is to automate the various bits of preprocessing that are typically done to RAPIDS package metadata prior to publishing packages.
-This includes the following notable changes:
-- Running [`rapids-dependency-file-generator`](https://github.com/rapidsai/dependency-file-generator) to get the dependencies for the CUDA version and architecture.
+
+It is responsible for the following:
+
+- Determining the correct dependencies for the package, based on the target CUDA version and architecture
+  - *by running [`rapids-dependency-file-generator`](https://github.com/rapidsai/dependency-file-generator)*
 - Modifying the package name to include a CUDA suffix (e.g. `"rmm" -> "rmm-cu11"`)
 - Updating the git commit embedded in the importable package.
 
-Since some of these modifications are only desirable in certain scenarios (wheel vs conda builds vs editable installs), all of these functions are customizable via the project's configuration in pyproject.toml.
-In cases where more dynamic customization is sensible, suitable environment variables and `config_settings` are supported during builds of distributions.
+## Support builders
```

Comment:
```suggestion
## Supported builders
```

Reply:
no no see they build support, you know like load-bearing.... ok yes this was a typo, thank you 🙈